src/components/ListComponent.js

```diff
@@ -36,6 +37,8 @@ const getComponent = (query, horizontal, sectionByDate) => {
     case QUERY_TYPES.VOLUNTEER.CALENDAR_ALL:
     case QUERY_TYPES.VOLUNTEER.CALENDAR_ALL_MY:
       return sectionByDate ? EventList : VerticalList;
+    case QUERY_TYPES.VOUCHERS:
+      return VouchersList;
```

Comment:
please compare with the other lists and use `VoucherList` with singular

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/988/commits/f34489f5029a9c350261281d39b8ff96d757ac1f#diff-f45f3db456082a41bc1ec4aff63c12fc88eb116b4ebb9fb374c4beb64bc35187